dev/Repeater/ItemsRepeater.idl

```diff
@@ -106,6 +106,9 @@ unsealed runtimeclass ItemsSourceView: Windows.UI.Xaml.Interop.INotifyCollection
     Boolean HasKeyIndexMapping{ get; };
     String KeyFromIndex(Int32 index);
     Int32 IndexFromKey(String key);
+
+    [WUXC_VERSION_PREVIEW]
+    Int32 IndexOf(IInspectable item);
```

Comment:
>IndexOf [](start = 10, length = 7)

I think we internally cast the ItemsSourceView to InspectingDataSource to get access to the IndexOf method. Can you do a search and remove those? now that this is public, we can just use it without casting.

Reply:
I've switched the areas in NavigationView that used that casting pattern to use IndexOf to just use the new method. However in the RadioButtons case, it turns out, that uppon initial loading, they crash when using the ItemsSourceView method.

Would you take a look if the way I implemented the IndexOf function is the reason for that?
From debugging, it seems that calling the virtual method ends up being the problem here.

Reply:
I'm sorry - I think we will have to wait on removing the casts until the API is in stable release, otherwise in stable releases the API still won't be available.

Reply:
Should I revert my latest commit, and we add that function as preview now or should we wait until the spec has gone through review?

Reply:
Yes. I think that is the best option for now. Thanks.

Reply:
Done!

Reply:
Yes. Thats a good idea.